From: Nicholas Krause <xerofoify@gmail.com>
Date: Sun, 5 Jul 2015 06:34:05 +0000
Subject: [PATCH] libceph: remove the unused macro AES_KEY_SIZE
Git-commit: b79b23682a1649f30960fb5bd920ba46c89a1b14
Patch-mainline: v4.3
References: Fate#318586


This removes the no longer used macro AES_KEY_SIZE as no functions use
this macro anymore and thus this macro can be removed due it no longer
being required.

Signed-off-by: Nicholas Krause <xerofoify@gmail.com>
Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 net/ceph/crypto.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/net/ceph/crypto.c
+++ b/net/ceph/crypto.c
@@ -78,10 +78,6 @@ int ceph_crypto_key_unarmor(struct ceph_
 	return 0;
 }
 
-
-
-#define AES_KEY_SIZE 16
-
 static struct crypto_blkcipher *ceph_crypto_alloc_cipher(void)
 {
 	return crypto_alloc_blkcipher("cbc(aes)", 0, CRYPTO_ALG_ASYNC);
